docs/sts/ldap.md

```diff
@@ -228,3 +231,35 @@ $ go run ldap.go -u foouser -p foopassword
         "sessionToken": "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJhY2Nlc3NLZXkiOiJOVUlCT1JaWVRWMkhHMkJNUlNYUiIsImF1ZCI6IlBvRWdYUDZ1Vk80NUlzRU5SbmdEWGo1QXU1WWEiLCJhenAiOiJQb0VnWFA2dVZPNDVJc0VOUm5nRFhqNUF1NVlhIiwiZXhwIjoxNTM0ODk2NjI5LCJpYXQiOjE1MzQ4OTMwMjksImlzcyI6Imh0dHBzOi8vbG9jYWxob3N0Ojk0NDMvb2F1dGgyL3Rva2VuIiwianRpIjoiNjY2OTZjZTctN2U1Ny00ZjU5LWI0MWQtM2E1YTMzZGZiNjA4In0.eJONnVaSVHypiXKEARSMnSKgr-2mlC2Sr4fEGJitLcJF_at3LeNdTHv0_oHsv6ZZA3zueVGgFlVXMlREgr9LXA"
 }
 ```
+
+## Managing User/Group Access Policy
+
+Access policies may be configured on a group or on a user directly. Access
+policies are first defined on the MinIO server using IAM policy JSON syntax. The
+`mc` tool is used to issue the necessary commands.
+
+**Note that by default no policy is set on a user**. Thus even if they
+successfully authenticate with AD/LDAP credentials, they have no access to
+object storage as the default access policy is to deny all access.
+
+To define a new policy, you can use the [AWS policy
+generator](https://awspolicygen.s3.amazonaws.com/policygen.html). Copy the
+policy into a text file `mypolicy.json` and issue the command like so:
+
+```shell
+mc admin policy add myminio mypolicy mypolicy.json
+```
+
+To assign the policy to a user or group, use:
+
+```shell
+mc admin policy set myminio mypolicy user=james
+
+mc admin policy set myminio mypolicy group=bigdatausers
+```
+
+**Please note that when AD/LDAP is configured, MinIO will not support long term
+users defined internally.** Only AD/LDAP users are allowed. In addition to this,
+the server will not support operations on users or groups using `mc admin user`
+or `mc admin group` commands. This is because users and groups are defined
```

Comment:
how does an admin know what policies are applied to a user or a group? Shouldn't we allow atleast info commands?

Reply:
That is being added in another PR #8307

Reply:
The right is to enable the `mc admin user info` and `mc admin group info` commands for LDAP configured MinIO as well. In the first command, the mc user provides a LDAP username to get their info. This would display any user policy directly attached as well list any groups the user is a member of. The `mc admin group info` command can return the policy associated with the user. Without LDAP configured, this command also returns a list of members, but with LDAP we will return a special value like "unknown" as we cannot produce a complete list.

If this plan sounds ok, I can send a PR for this. Thoughts?

Reply:
Sounds good to me

Reply:
@donatello the PR here #8307 adds similar functionality. IF we are going to bring it in we should bring the change you proposed also in the same PR, It may allow for no breaking changes in admin API.